analyzers/its/expected/Nancy/Nancy--net452-S2930.json

```diff
@@ -0,0 +1,17 @@
+{
+"issues":  [
+{
+"id":  "S2930",
+"message":  "Dispose 'engineDisposedCts' when it is no longer needed.",
+"location":  {
+"uri":  "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/Nancy/src/Nancy/NancyEngine.cs#L98",
```

Comment:
This is a FP, as it is disposed in dispose method and we can't go cross-procedure